language/sample-apps/README.md

```diff
@@ -0,0 +1,38 @@
+# Sample Applications to accelerate your Gen AI Applications on Google Cloud
+
+Looking to build Gen AI applications that integrate with Vertex AI PaLM Foundational models? Would you like to use utilize standard frameworks like Python Flask, Streamlit, Gradio to host these applications on Google Cloud? You have come to the right place.  
+
+The Sample Applications listed are provided as application templates that you can use. The key goal of these applications is to get you started quickly and help you understand how you can integrate the Vertex PaLM API and the necessary commands to deploy these applications to Google Cloud. 
+
+You can go through the various applications and pick an application or two that interests you. Click on any of the applications to see detailed documentation, sample template and instructions to deploy on Google Cloud.
+
+## Environment Setup
+
+We provide instructions for setting up your environment in [Cloud Shell](https://cloud.google.com/shell). Before you run any of the sample applications, ensure that you have followed the instructions in [SETUP.md](SETUP.md).
+
+## Sample Applications
+
+| Requirement | Application Name | Technologies Used |
+|---|---|---|
+|Develop a chat application using Python Flask framework and Vertex AI PaLM API model |[chat-flask-cloudrun](chat-flask-cloudrun)|Cloud Run, Python Flask|
+|Develop a chat application using [Gradio](https://www.gradio.app/) framework and Vertex AI PaLM API model|[chat-gradio](chat-gradio)|Cloud Run, Gradio, Python|
+|Develop a chat application using [Streamlit](https://streamlit.io/) framework and Vertex AI PaLM API model|[chat-streamlit](chat-streamlit)|Cloud Run, Streamlit, Python|
+|Provide an API for the Vertex AI PaLM Code Model for your client applications.|[code-predict-cloudfunction](code-predict-cloudfunction)|Cloud Functions v2, Python|
+|Provide an API for the Vertex AI PaLM Text Model for your client applications.|[text-predict-cloudfunction](text-predict-cloudfunction)|Cloud Functions v2, Python|
+|Develop an Event-Driven application that processes uploaded files and summarizes their content|[summarization-gcs-cloudfunction](summarization-gcs-cloudfunction) |Cloud Functions v2, Cloud Storage, Python|
+
```

Comment:
Could you also add an external link to the JSS solution
https://cloud.google.com/architecture/ai-ml/generative-ai-document-summarization

Reply:
Added the link to the solution.